test/gtest/uct/ib/test_ib.cc

```diff
@@ -404,7 +404,7 @@ class test_uct_ib_gid_idx : public test_uct_ib_with_specific_port {
     }
 };
 
-UCS_TEST_P(test_uct_ib_gid_idx, non_default_gid_idx, "GID_INDEX=1") {
+UCS_TEST_P(test_uct_ib_gid_idx, non_default_gid_idx, "IB_GID_INDEX=1") {
```

Comment:
I think we do not need to change UCT tests because the UCT layer is not affected by UCP config parser change

Reply:
actually we do need because uct_config_modify is changed to pass prefix to config_parser